### ISSUE TITLE
automatically suggest app if no window was found

### ISSUE THREAD

**Issue by wbolster** [daniellandau/switcher#31]
when i am looking for a window, but there is no match, i would like to see matching apps instead, since the next step is usually launching that app.

**Comment by daniellandau** [comment]
I've been thinking that maybe instead of, or in addition to the current keyboard shortcuts for switching mode also the return key could switch when there are no results. Would that do it for you?

**Comment by wbolster** [comment]
not entirely convinced. :)

in general, the best interaction pattern is no interaction at all. empty lists are useless.

**Comment by daniellandau** [comment]
True, but also surprises are bad. I'll code both of these and see what I end up liking (and maybe make it configurable).

(Btw, huge thanks and good stuff in this and the other issues you just filed!)

**Comment by wbolster** [comment]
yw, keep up the good work!

 i only get vocal about things i care about. this extension is a big improvement to my gnome desktop experience. i would love to make it even better. :)

**Comment by daniellandau** [comment]
Fixed in https://github.com/daniellandau/switcher/commit/26ff2af31f5ffc862a28172d9bc4c994df105558.
I ended up with the solution where the mode is switched to the launcher if there are zero matching in the current mode and more than zero matching in the other mode. That way a typo like "friefox" doesn't just put you in the launcher mode but typing "thunderbird" when you don't have it open allows opening Thunderbird quickly.
